content/blog/empty-psalm-87-firm-foundation.md

```diff
@@ -0,0 +1 @@
+﻿---
title: "Empty: Psalm 87 - Firm Foundation"
image: "https://images.unsplash.com/photo-1532585825435-07e962c2eaf1?crop=entropy&cs=srgb&fm=jpg&ixid=Mnw5NjYxfDB8MXxzZWFyY2h8Nnx8YmVkcm9ja3xlbnwwfHx8fDE2MTY2ODU2NTc&ixlib=rb-1.2.1&q=85"
date: "2021-03-25T10:14:20.000Z"
author:
  id: "trevor-chadwick"
  name: "Trevor Chadwick"
  role: "Research Team"
topics:
  - "devotionals"
---
[**Psalm 87**][psa87]

Any architect realizes that a firm foundation is crucial to any structure. Once there is a firm foundation, supports are put in, allowing the rest of the building to be built. We can see, from the first verse in Psalm 87, that God knows (obviously) how vital a firm foundation is. Of all the places in the world, He set His earthly temple upon the mountains. Now, if we dig a little deeper into this and look at [Isaiah 28:16][isa2816], we see that He was also planning on laying another foundation in Zion. This foundation is the foundation of the church which cannot sink or totter. This foundation is our Rock, Jesus Christ, which God has laid.

Storms are an inescapable part of life. Honestly, everybody will experience a train wreck or two, and they always happen at the worst times. Nobody saw COVID-19 coming, but now it has completely changed many of our lives in ways we could never even dream of. It has shaken some of us to the very core and exposed the true nature and durability of the foundations some of us have been building our lives upon, such as: our own careers, our own 5- to 10-year plans, our businesses, our financial investments, and even politicians. Jesus warns us against building our lives on things of the world ([Matthew 7:24-27][mat7]). A life built on worldly virtues or morals is not solid.

If we want to know for sure that we can endure life’s greatest storms, no matter how much it rains or how strong the winds blow, we have to build on the bedrock of lasting truth, Jesus Christ. It will take time and effort to build this way, but trust Jesus when He says it’s worth it. Hearing of Jesus’ sacrifice on the cross gives us the long-lasting peace that we are forgiven. Hearing that Jesus rose and ascended to heaven to rule over all things offers us firm hope during the uncertain times in life. Don’t build your life on sinking sand. Build on Jesus Christ, the solid rock. His teachings provide us with the strongest foundation. Confidently build your life on Him.

[psa87]: https://biblehub.com/psalms/87.htm
[isa2816]: https://biblehub.com/isaiah/28-16.htm
[mat7]: https://biblehub.com/context/matthew/7-24.htm

```

Comment:
Hey looks like something got screwy here. These look like some sort of escape character. Mind cleaning these up?